locations/spiders/disco_argentina.py

```diff
@@ -6,7 +6,7 @@
 
 class DiscoArgentinaSpider(Spider):
     name = "disco_argentina"
-    item_attributes = {"brand": "Disco Argentina"}
```

Comment:
This is different to the brand in NSI.
https://nsi.guide/index.html?t=brands&k=shop&v=supermarket&tt=Q6135978#disco-0d2b84